releasenotes/notes/no_warning_with_reverse_bits-b47cb1e357201593.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    When :func:`~circuit_drawer` was used with `reverse_bits=True` on a circuit without classical bits, warning about `cregbundle` was raised, creating confusion. The warning was removed for this case.
```

Comment:
This line could do with wrapping - when reno generates the rst for the metapackage, this goes in at a few levels of nesting that might cause its linter to complain.

(It wouldn't hurt for us to get a checker running on Terra to enforce that.)

Reply:
Fixed in 56d3e384de1669086f26738401e738eaef2b71ca